app/submitter/converter.py

```diff
@@ -76,6 +77,13 @@ def convert_answers(schema, questionnaire_store, routing_path, flushed=False):
         'questionnaire_id': metadata['questionnaire_id'],
     }
 
+    data = {
```

Comment:
Since this is only used for `0.0.3` this could be moved under `if schema.json['data_version'] == '0.0.3':` so we only create it when needed.

Reply:
We've added it now, thanks